template.html

```diff
@@ -14,8 +14,8 @@ <h1>Julia Hacktoberfest</h1>
 <div id="resources">
     <h2>Useful Resources</h2>
     <ul>
-        <li><a target="_blank" rel="noopener" href="https://hacktoberfest.digitalocean.com/">Hacktoberfest 2020</a> by DigitalOcean</li>
         <li><a target="_blank" rel="noopener" href="https://kshyatt.github.io/post/firstjuliapr/">Making a first Julia pull request</a> by <a target="_blank" rel="noopener" href="https://twitter.com/kslimes">Katharine Hyatt</a></li>
+        <li><a target="_blank" rel="noopener" href="https://github.com/miguelraz/StartHere.jl">StartHere.jl</a> by <a target="_blank" rel="noopener" href="https://twitter.com/miguelraz_">Miguel Raz</a></li>
```

Comment:
@miguelraz could you check if this info is correct? :)

Reply:
No! My full name/surname is "Miguel Raz Guzmán". Thanks for checking though :dancers:

Reply:
```suggestion
        <li><a target="_blank" rel="noopener" href="https://github.com/miguelraz/StartHere.jl">StartHere.jl</a> by <a target="_blank" rel="noopener" href="https://twitter.com/miguelraz_">Miguel Raz Guzmán</a></li>
```

Reply:
Sorry! Fixed it